mobile/android/android-application-manager.ts

```diff
@@ -27,6 +29,16 @@ export class AndroidApplicationManager extends ApplicationManagerBase {
 		}).future<string[]>()();
 	}
 
+	public getApplicationsLiveSyncSupportedStatus(newApplications: string[]): IFuture<void> {
+		return (() => {
+			let liveSyncStatus = getFuturesResults<boolean>(_.map(newApplications, app => this.isLiveSyncSupportedForApplication(app)), () => true);
+			this.applicationsLiveSyncStatus = (this.applicationsLiveSyncStatus || [])
+												.concat(_.map(newApplications, (app: string, index: number) =>
```

Comment:
In case of renaming `newApplications`, the `app` parameter in the lambda function should also be renamed